src/core/sentinel.wat

```diff
@@ -0,0 +1,28 @@
+(module
+  (type $empty_struct (struct))
+  (import "env" "is_safari" (global $is_safari i32))
+  (import "env" "safari_error" (global $safari_error externref))
+  (import "env" "safari_is_error" (func $safari_is_error (param externref) (result i32)))
```

Comment:
Could you please cross-reference `emscripten-settings.ts` to know where these variables come from?